portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,18 @@
   width: 650px;
 }
 
-#greeting-container {
+img {
```

Comment:
Use an ID?

Reply:
Sounds good! I just submitted a commit to this PR hat should change add an id to this